mysql-plugin/docs/Mysql-batchsource.md

```diff
@@ -85,14 +85,14 @@ with the tradeoff of higher memory usage.
 Data Types Mapping
 ----------
 | MySQL Data Type                | CDAP Schema Data Type | Comment                                                 |
-| ------------------------------ | --------------------- |---------------------------------------------------------|
+| ------------------------------ |-----------------------|---------------------------------------------------------|
 | BIT                            | boolean               |                                                         |
 | TINYINT                        | int                   |                                                         |
 | BOOL, BOOLEAN                  | boolean               |                                                         |
 | SMALLINT                       | int                   |                                                         |
-| MEDIUMINT                      | double                |                                                         |
+| MEDIUMINT                      | int                   | Users can manually set output schema to map it to Long. |
```

Comment:
should Long in the comment be double? Or was the previous 'double' type a documentation bug?

Reply:
It was a documentation bug. MediumInt used to get detected as int and Mediumint unsigned as long.